### ISSUE TITLE
ncurses problems

### ISSUE THREAD

**Issue by justinwoo** [justinwoo/easy-purescript-nix#188]
https://github.com/justinwoo/easy-purescript-nix/runs/5388418344?check_suite_focus=true#step:4:1836

```
purs: /nix/store/j0ibsi9cwb1aaglxmfbzvf18gqw0ylri-ncurses-6.2-abi5-compat/lib/libtinfo.so.5: no version information available (required by purs)
0.14.7
```

@toastal do you happen to know why purescript releases have ncurses problems now?

**Comment by justinwoo** [comment]
happens in 0.14.6 and 0.14.7

**Comment by toastal** [comment]
I don't. I don't really follow what the core team does at that closely, but I know `ncurses` has been an issue in the past and with different Linux distros

**Comment by JordanMartinez** [comment]
See https://github.com/purescript/purescript/issues/4253

**Comment by jamesdbrock** [comment]
This is fixed in https://github.com/NixOS/nixpkgs/pull/138555/files

Here is a temporary fix if you're still on nixpkgs 21.11.

```nix
  nixpkgs.overlays = [
    (self: super: {
      ncurses5 = super.ncurses5.overrideAttrs (attr: {
        configureFlags = attr.configureFlags ++ ["--with-versioned-syms"];
      });
    }) 
  ];
```

https://discord.com/channels/864614189094928394/869252844514074644/951838088588591124
